ironfish/src/consensus/__fixtures__/verifier.test.ts.fixture

```diff
@@ -1040,5 +1040,375 @@
       "type": "Buffer",
       "data": "base64:AAAAAAAAAAABAAAAAAAAAABsyoj/////AAAAAJQMIVcNrLrG5INZFrJTSocL6mK0JGv1xJavAt+ZnaUyGAUfKiPO4Wai6iRNf2pbNZIxYgyb3juuORpIPeIpcwFZA/vDiyX0h4xHNJTnsiVLejv/CTqZbDTT7q32G+ZpNxkbodUBgYq7J+H5RhUpNBJ8cU/Fygdam6aBHcZ5eadwtBNT2aVXGGI/3ZFVPlJCdrkn2+WfXGvK2X/JdF+4lE8OogIoXDVDPCedYY/rfFIUoYwfGwphmEqbR3hIWWYNuNX0i/pBGczEBk/D8zPSZ+gzYo8UdDjav0Q/HCnzRmrer9WlaM1Ls8CXKhfT4mXDqEGqs0uMvpYtqzpeDrID0yjnhVvtgXkyLcf/VwwUk2VfP/oUB/NSKufV6goAr7voAwzebje/WBKorAmgyPJsw0GgaTwS0x0zJnhFfw63ePvrlJTHJpF4l+TQptTx3Mre8oehNLALrVfVqx/EWvhf49hVgsirqy4qIdpTMD7AMEBsZAcW0qeaCsN3uEtkbQdgeV0SrEJlYW5zdGFsayBub3RlIGVuY3J5cHRpb24gbWluZXIga2V5MDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAw3YfS5UJYhIlVI5TGg5dSsDi2onGhDMb1F0Uo26l59tSwMwLEN2QCt4vZirf4kugxPw4iULF+OLdVi5VPXytYDg=="
     }
+  ],
+  "Verifier verifyConnectedSpends says the block with spends is valid": [
```

Comment:
Should there be lines deleted out of this fixture? It looks like since it's just a rename, there should be as many deleted lines as additional lines.

Reply:
Removed old fixtures